packages/app-store/make/pages/setup/index.tsx

```diff
@@ -0,0 +1,174 @@
+import { Trans } from "next-i18next";
+import Link from "next/link";
+import { useState } from "react";
+import { Toaster } from "react-hot-toast";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc/react";
+import { Button, Tooltip, showToast } from "@calcom/ui";
+import { Clipboard } from "@calcom/ui/components/icon";
+
+const MAKE = "make";
+const appInviteLink = process.env.MAKE_INVITE_LINK; //TODO Add MAKE_INVITE_LINK to env vars
+
+export default function MakeSetup() {
```

Comment:
nit: This is almost identical with Zapier. I think it would make sense to have a common component used by both
![image](https://github.com/calcom/cal.com/assets/1780212/892d0c0c-5377-4f33-a22f-b42940267481)

We can have something like `ApiKeysSetup` that would take care of the following UI
![image](https://github.com/calcom/cal.com/assets/1780212/a63eddcd-be9d-42b8-a782-62c352859440)

The instructions can be kept per app

Reply:
where would you add this component?

Reply:
I guess app-store/_components would be the right place for now as I don't see any usage as of now across the webapp.

Reply:
will open another PR for this